src/app/header/header.component.html

```diff
@@ -65,23 +66,47 @@ <h5 id="offcanvasStartLabel">penda</h5>
                     <a
                         href="https://github.com/OpenCovenant"
                         role="button"
-                        class="list-group-item list-group-item-action mb-3"
+                        class="navText list-group-item-action mb-3"
                         target="_blank"
                         rel="noopener noreferrer"
-                        >Rreth&nbsp;Nesh</a
+                        ><i class="bi bi-house-door"></i> Rreth&nbsp;Nesh</a
                     >
                 </div>
                 <div>
                     <a
                         href="https://github.com/OpenCovenant/ink"
                         role="button"
-                        class="list-group-item list-group-item-action mb-3"
+                        class="navText list-group-item-action mb-3"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        ><i class="bi bi-info-circle"></i>Rreth&nbsp;Penda</a
+                    >
+                </div>
+                <hr />
+                <div>
+                    <a
+                        href="#"
+                        role="button"
+                        class="navText list-group-item-action mb-3"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        ><i class="bi bi-gear"></i>Settings&nbsp;</a
+                    >
+                </div>
+
+                <div class="">
```

Comment:
AFAIK this is redundant, i.e. `class=""` doesn't do much, so I guess it probably should be removed?